### DISCUSSION TITLE
Replicated SQLite

### DISCUSSION THREAD

**Discussion by gquintana** [grafana/grafana#39987] [General]
Storing data in SQLite is a very good feature. As long as the data volume is moderate, having to set up a replicated MySQL or PostgreSQL for the sake of getting [https://grafana.com/docs/grafana/latest/administration/set-up-for-high-availability/](Grafana High Availability) is very complicated.

I wonder whether it would be possible to plug Grafana on a replicated SQLite, either:

- [https://github.com/canonical/go-dqlite](Canonical DQLite)
- [https://github.com/rqlite/rqlite](RQLite)
- [http://litereplica.io](LiteReplica)

Is Grafana SQL connection generic enough to be able to plug new databases?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
